units/building_container_image.go

```diff
@@ -147,3 +159,26 @@ func (j *buildingContainerImageJob) Run(ctx context.Context) {
 		return
 	}
 }
+
+func (j *buildingContainerImageJob) tryRequeue(ctx context.Context) {
+	if j.shouldRetry(ctx) && j.env.RemoteQueue().Started() {
+		job := NewBuildingContainerImageJob(j.env, j.parent, j.DockerOptions, j.Provider)
+		job.UpdateTimeInfo(amboy.JobTimeInfo{
+			WaitUntil: time.Now().Add(time.Second * 10),
+		})
+		err := j.env.RemoteQueue().Put(job)
+		grip.Error(message.WrapError(err, message.Fields{
+			"message":   "failed to requeue setup job",
+			"operation": "container build",
+			"host":      j.ParentID,
+			"job":       j.ID(),
+			"attempts":  j.parent.ContainerBuildAttempt,
```

Comment:
Does this get incremented somewhere?

Reply:
Yes: IncContainerBuildAttempt() gets called in the defer function in Run().